mypyc/ir/class_ir.py

```diff
@@ -237,6 +237,16 @@ def __repr__(self) -> str:
     def fullname(self) -> str:
         return f"{self.module_name}.{self.name}"
 
+    @property
+    def allow_interpreted_subclasses(self) -> bool:
```

Comment:
I'm not entirely sure that this is correct in terms of the rest of the codebase right now, but semantically it feels like the correct thing to do